src/components/modules/OtpCode/OtpCode.view.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <form @submit.prevent="onSubmit">
+    <v-title>
+      {{ $t('components.otp.title') }}
+    </v-title>
+    <v-description>
+      {{ $t('components.otp.description') }}
+    </v-description>
+
+    <form-item>
+      <v-input
+        v-model="code"
+        v-validate="'required|digits:6'"
+        data-vv-as="code"
+        data-vv-name="code"
+        :error="errors.first('code') || error"
+        name="code"
+        :placeholder="$t('components.otp.enterReceivedCode')"
+        data-test="code-input"
+      />
+    </form-item>
+    <form-item class="v-mb-24">
+      <v-button
+        :disabled="!isFormValid || isLoading"
+        :is-loading="isLoading"
+        type="submit"
+        data-test="submit-button"
+      >
+        {{ $t('global.confirm') }}
+      </v-button>
+    </form-item>
+    <form-row class="v-fs-14 v-text-center">
+      <v-link
+        :disabled="isLoading"
+        href="#"
+        data-test="recovery-link"
+        @click.prevent="onRecover"
+      >
+        {{ $t('components.otp.noCode') }}
+      </v-link>
+    </form-row>
+  </form>
+</template>
+
+<script>
+import VButton from '@endpass/ui/kit/VButton';
+import VInput from '@endpass/ui/kit/VInput';
+import VLink from '@endpass/ui/kit/VLink';
+import FormItem from '@/components/common/FormItem';
+import FormRow from '@/components/common/FormRow';
+import formMixin from '@/mixins/form';
+import VTitle from '@/components/common/VTitle';
+import VDescription from '@/components/common/VDescription';
+
+export default {
+  name: 'OtpCodeView',
+
+  props: {
+    isLoading: {
+      type: Boolean,
+      default: false,
+    },
+  },
+
+  data: () => ({
+    code: '',
+  }),
+
+  methods: {
+    onSubmit() {
+      this.$emit('submit', { code: this.code });
+    },
+
+    onRecover() {
+      this.$emit('recover');
+    },
+  },
+
+  mixins: [formMixin],
+
+  components: {
+    VTitle,
+    VDescription,
+    VLink,
+    VButton,
+    VInput,
+    FormItem,
+    FormRow,
+  },
+};
+</script>
+
+<style lang="postcss"></style>
```

Comment:
Do we migrate to `postcss`? :D

Reply:
It was there before me :slightly_smiling_face: